src/platforms/eglstream-kms/server/display.cpp

```diff
@@ -403,11 +486,11 @@ void mge::Display::configure(DisplayConfiguration const& conf)
 {
     auto kms_conf = dynamic_cast<KMSDisplayConfiguration const&>(conf);
     active_sync_groups.clear();
-    kms_conf.for_each_output([this](kms::EGLOutput const& output)
+    kms_conf.for_each_output([this](std::shared_ptr<kms::EGLOutput> const& output)
```

Comment:
Shouldn't `for_each_output()` keep the same signature and dereference the `shared_ptr<>`? This is exposing an implementation detail

Reply:
On closer examination, ownership **is** being passed this way